lib/appsignal/config.ex

```diff
@@ -56,9 +55,7 @@ defmodule Appsignal.Config do
 
     config =
       config
-      |> merge_filter_data_keys(Application.get_env(:phoenix, :filter_parameters, []))
-      |> merge_filter_data_keys(config[:filter_parameters])
-      |> merge_filter_data_keys(config[:filter_session_data])
+      |> merge_filter_parameters(Application.get_env(:phoenix, :filter_parameters, []))
```

Comment:
No merging for `filter_session_data` here?

Reply:
No, we're not supposed to merge them all into one option anymore. Otherwise session data filter keys would also be filtered out in parameters.

This only merges the phoenix denylist keys into the AppSignal filter_parameters config option.